pubspec.yaml

```diff
@@ -17,16 +17,16 @@ dependencies:
   path: ^1.8.2
   win32: ^5.5.1
   cross_file: ^0.3.3+7
-  web: ^0.5.1
+  web: ^1.0.0
 
 dev_dependencies:
-  lints: ^3.0.0
+  lints: ^4.0.0
```

Comment:
One of the other PR's for this issue bumped `lints` to 4.0.0, which is currently the latest version. I did the same thing here for consistency and better coverage of lints.